### ISSUE TITLE
Add plugin button placement

### ISSUE THREAD

**Issue by Farmer-Eds-Shed** [symbioquine/farmOS_asset_link#4]
Green add plugin button covers Kebab menu button of last plugin making selecting it difficult, especially on Laptop/Desktop browsers.
![image](https://user-images.githubusercontent.com/63633329/189473580-f5bfa561-7039-47a6-9ed3-2bfd21164ab4.png)

**Comment by Farmer-Eds-Shed** [comment]
I see it's actually simple to change since the manager is a plugin.
line 58 in ManagePlugins.alink.vue changed to
`    <q-page-sticky position="bottom-right" :offset="[45, 30]">`
looks more acceptable to me in both mobile and on the desktop browser, by moving up and left allows better access to menu and the last plugin can be clearly read.
![image](https://user-images.githubusercontent.com/63633329/189474584-6c0b1889-2e5e-4959-9b7a-2f21d64968cc.png)
![image](https://user-images.githubusercontent.com/63633329/189474685-091ed543-e386-4431-baaa-c92ee84bb53b.png)

**Comment by symbioquine** [comment]
@Farmer-Eds-Shed Thanks for opening this! That has been bugging me for a while now too.

I'm not sure I'm a fan of bumping the button in from the right like that though...

What do you think about this strategy instead?

```patch
diff --git a/alinkjs/packages/assetlink-default-plugins/plugins/ManagePlugins.alink.vue b/alinkjs/packages/assetlink-default-plugins/plugins/ManagePlugins.alink.vue
index a83a493..284ac05 100644
--- a/alinkjs/packages/assetlink-default-plugins/plugins/ManagePlugins.alink.vue
+++ b/alinkjs/packages/assetlink-default-plugins/plugins/ManagePlugins.alink.vue
@@ -6,6 +6,7 @@
       node-key="label"
       default-expand-all
       dense
+      class="q-mb-xl q-pb-md"
     >
       <template v-slot:default-header="prop">
         <div class="row items-center" style="width: 100%">
```

That adds some extra bottom spacing to the tree such that it is always possible to scroll to the point where the floating action button lands below the last item.

![image](https://user-images.githubusercontent.com/30754460/189486347-58eae6c4-b9d1-4ec5-9548-520963f2e3f5.png)

**Comment by Farmer-Eds-Shed** [comment]
Yea, that works for me too.

**Comment by symbioquine** [comment]
Cool

**Comment by symbioquine** [comment]
This is fixed in [release 0.7.2](https://www.drupal.org/project/farmos_asset_link/releases/0.7.2) by https://github.com/symbioquine/farmOS_asset_link/commit/0bb94a304fb8868234d2d7fcb8f9ecff1a7dbe11
